Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -712,7 +712,7 @@ def setUp(self):
             self.set_event_loop(self.loop)
 
 
-class GenericWatcherTests:
+class GenericWatcherTests(test_utils.TestCase):
```

Comment:
this test wasn't enabled?

Reply:
I split this fix out, so this PR is easier to review: https://github.com/python/cpython/pull/95009